weblate/machinery/tests.py

```diff
@@ -378,14 +380,15 @@ def test_microsoft_cognitive(self):
         )
         responses.add(
             responses.GET,
-            'https://api.microsofttranslator.com/V2/Ajax.svc/'
-            'GetLanguagesForTranslate',
+            'https://api.cognitive.microsofttranslator.com/'
+            'languages?api-version=3.0',
             json=["en", "cs"],
         )
         responses.add(
-            responses.GET,
-            'https://api.microsofttranslator.com/V2/Ajax.svc/Translate',
-            json="svět",
+            responses.POST,
+            'https://api.cognitive.microsofttranslator.com/'
+            'translate?api-version=3.0&from=en&to=cs&category=general',
+            json=MICROSOFT_RESPONSE,
         )
```

Comment:
Can you please add second test where you would override region as well?  It should use different URL for issuing token.

Reply:
Sure, thanks for pointing out.